packages/babel-preset-env/test/normalize-options.spec.js

```diff
@@ -191,6 +192,44 @@ describe("normalize-options", () => {
     });
   });
 
+  describe("validateIncludeExcludeOption", () => {
+    it("should not throw if supplied include option is array ", () => {
+      expect(() => {
+        validateIncludeExcludeOption(["transform-spread"], "include");
+      }).not.toThrow();
+    });
+
+    it("should not throw if include option is undefined ", () => {
+      expect(() => {
+        validateIncludeExcludeOption(["transform-spread"], "include");
```

Comment:
Should this be `undefined` instead of an array?

Reply:
Yeah, will fix it now, thanks.